pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/OfflineClusterServerQueryKillingTest.java

```diff
@@ -56,10 +56,10 @@
 
 
 /**
- * Integration test that converts Avro data for 12 segments and runs queries against it.
+ * Integration test for heap size based server query killing, this works only for xmx4G
  */
-public class OfflineClusterQueryKillingTest extends BaseClusterIntegrationTestSet {
-  private static final Logger LOGGER = LoggerFactory.getLogger(OfflineClusterQueryKillingTest.class);
+public class OfflineClusterServerQueryKillingTest extends BaseClusterIntegrationTestSet {
```

Comment:
May want to rename to `OfflineServerMemBasedQueryKilingTest`

Reply:
fixed